src/definitions/modules/dropdown.less

```diff
@@ -565,13 +565,23 @@ select.ui.dropdown {
     padding: @selectionItemPadding;
     white-space: normal;
     word-wrap: normal;
+    z-index: 0;
```

Comment:
In relation to my below `@supports` suggestion, we should also put this into the extra wrapper and add it to the `:hover` selector only. This way, there won't be any possible sideeffects